packages/framework/src/DataFixtures/DemoMultidomain/ArticleDataFixture.php

```diff
@@ -74,11 +96,11 @@ public function load(ObjectManager $manager)
      * @param \Shopsys\FrameworkBundle\Model\Article\ArticleData $articleData
      * @param string|null $referenceName
      */
-    private function createArticle(ArticleData $articleData, $referenceName = null)
+    private function createArticle(ArticleData $articleData, string $referenceName = null)
```

Comment:
I didn't realize this, but luckily it makes sense, because the function is called with only one parameter. If I changed the signature to `?string`, then calling without second parameter would fail